core/src/main/java/org/infinispan/interceptors/impl/InvocationContextInterceptor.java

```diff
@@ -125,9 +126,10 @@ private void rethrowException(InvocationContext ctx, VisitableCommand command, T
             log.debug("Exception executing call", th);
          } else if (th instanceof OutdatedTopologyException) {
             if (log.isTraceEnabled()) log.tracef("Topology changed, retrying command: %s", th);
+         } else if (DeadlockDetectedException.isDeadlockDetectedException(th)) {
```

Comment:
To avoid cluttering the logs. I am unsure if this is necessary because on real usage *we hope* there aren't that many deadlocks detected. However, in the reproducer I have, this causes too much noise.